README.markdown

```diff
@@ -280,7 +280,15 @@ $redis->swapdb(0, 1); /* Swaps DB 0 with DB 1 atomically */
 
 ### close
 -----
-_**Description**_: Disconnects from the Redis instance, except when `pconnect` is used.
+_**Description**_: Disconnects from the Redis instance, include when `pconnect` is used.
+
+*Note*: Requires Redis >= 4.2.0, phpredis can close the persistent connection.
```

Comment:
> Requires Redis >= 4.2.0, phpredis can close the persistent connection.

Redis >= 4.2.0 is required to close a persistent connection.

**However, I prefer:**

Closing a persistent connection requires Redis >= 4.2.0.